lib/properties/amount_of_substance.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:units_converter/models/conversion_node.dart';
+import 'package:units_converter/models/double_property.dart';
+import 'package:units_converter/models/property.dart';
+import 'package:units_converter/models/unit.dart';
+
+//Available length units
+enum AMOUNTOFSUBSTANCE {
```

Comment:
Prefer SCREAMING_SNAKE_CASE to be more coerent

Reply:
5a36c231ebe5eb099923287b7c01506607d7e638